### PR TITLE
Update Overview of APIs per December RelNotes for Release SDK

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article for review: 
* https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2975 - Find "incoming"

Outline of headings in the PR; where to move the Incoming sections?  Find "incoming" below:
```
## Incoming APIs Dec 2023 v120
#### 1. Support for browser extensions in WebView2 - put in Browser features section.
#### 2. Frame Process Info - put in Process management, or near "iframes".
#### 3. AcceleratorKeyPressedEventArgs.IsBrowserAcceleratorKeyEnabled - put near Browser features > Swipe gesture navigation.
#### 4. Managing profile deletion - put near User data > Multiple profiles.

# Overview of WebView2 features and APIs

## Main classes: Environment, Controller, and Core

## Web/native interop
#### Host/web object sharing
#### Script execution
#### Web messaging
#### Script dialogs
#### Shared buffer

## Browser features
#### Printing
#### Cookies
#### Image capture
#### Downloads
#### Permissions
#### Context menus
#### Status bar
#### User Agent
#### Autofill
#### Audio
#### Swipe gesture navigation
#### Enable or disable the browser responding to accelerator keys (shortcut keys) - NEW; incoming 3
#### Fullscreen
#### PDF toolbar
#### Theming
#### Language
#### New window
#### Close window
#### Document title
#### Favicon
#### Security and privacy
###### Tracking prevention
###### SmartScreen
###### Custom crash reporting
#### Browser extensions - NEW; incoming 1

## Process management
#### Frame process info - NEW; incoming 2

## Navigate to pages and manage loaded content
#### Manage content loaded into WebView2
#### Navigation history
#### Navigation kind
#### Block unwanted navigating
#### Navigation events
#### Manage network requests in WebView2
#### Custom scheme registration
#### Client certificates
#### Server certificates
#### Launch an external URI scheme

## iframes

## Authentication

## Rendering WebView2 in non-framework apps
#### Sizing, positioning, and visibility
#### Zooming
#### Rasterization scale
#### Focus and tabbing
#### Parent window
#### Keyboard accelerators
#### Default background color

## Rendering WebView2 using Composition
#### Connecting to the visual tree
#### Forwarding input
#### Drag and drop
#### Accessibility

## User data
#### Multiple profiles
#### Delete a profile - NEW; incoming 4

## Performance and debugging
#### Memory usage target

## Chrome DevTools Protocol (CDP)

## See also
```

AB#48156001

### CODE REVIEW

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 1]
```diff
@@ -836,6 +867,34 @@ This feature is currently disabled by default in the browser.  To enable this fe
...
 ---
 
+
+<!-- ------------------------------ -->
+#### Enable or disable the browser responding to accelerator keys (shortcut keys) (incoming 3)
+
+`ICoreWebView2AcceleratorKeyPressedEventArgs` has a `IsBrowserAcceleratorKeyEnabled` property to allow you to control whether the browser handles accelerator keys (shortcut keys), such as **Ctrl+P** or **F3**.
+
+See also:
+* [Keyboard accelerators](#enable-or-disable-the-browser-responding-to-accelerator-keys-shortcut-keys-incoming-3), below.
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:877]: I think this link might be a mistake. It links back to this section.

I'm assuming that you are intending to link it to a documentation about keyboard accelerators.

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 2]
```diff
@@ -1235,11 +1372,106 @@ Get information about running WebView2 processes, exiting processes, and failed
...
 
+<!-- todo: ~ move 2 to here -->
+<!-- ------------------------------ -->
+#### Frame process info (incoming 2)
+<!-- todo: maybe move to near "iframes" instead of in Process management -->
+
+The Frame Process Info API, including `GetProcessExtendedInfos`, provides a snapshot collection of all frames that are actively running in the associated renderer process.  This API enables your app to detect which part of WebView2 is consuming resources such as memory or CPU usage.
+
+See also:
+<!-- todo -->
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:1383]: this see also is blank. I think we can remove it?

**Review by victorhuangwq — CHANGES_REQUESTED** [review]
Adjustments regarding see also

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:1383]: removed

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:877]: fixed link
`* [Keyboard accelerators](#keyboard-accelerators), below.`

**Review by victorhuangwq — APPROVED** [review]
LGTM

**Review by peiche-jessica — APPROVED** [review]

**Pull request merged**